<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResetDialog</class>
 <widget class="QDialog" name="ResetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item alignment="Qt::AlignmentFlag::AlignHCenter">
    <widget class="QLabel" name="err_tip">
     <property name="text">
      <string>错误信息</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="user_label">
       <property name="text">
        <string>用户：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="user_lineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="email_label">
       <property name="text">
        <string>邮箱：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="email_lineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="ver_label">
       <property name="text">
        <string>验证码：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ver_lineEdit"/>
     </item>
     <item>
      <widget class="TimberBtn" name="get_code">
       <property name="text">
        <string>获取</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="newpwd_label">
       <property name="text">
        <string>新密码：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="newpwd_lineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="confirm_pushButton">
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancal_pushButton">
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TimberBtn</class>
   <extends>QPushButton</extends>
   <header>timberbtn.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
